src/test/java/com/openlattice/rehearsal/data/DataControllerTest.kt

```diff
@@ -1215,4 +1219,21 @@ class DataControllerTest : MultipleAuthenticatedUsersBase() {
         val loadedEdgeDstEntities2 = dataApi.loadEntitySetData(esEdgeDst2.id, essEdgeDst2, FileType.json).toList()
         Assert.assertEquals(0, loadedEdgeDstEntities2.size)
     }
+
+    @Test
+    fun testGetEntitySetSize() {
+        val et = createEntityType()
+        val es = createEntitySet(et)
+
+        val testData = TestDataFactory.randomStringEntityData(numberOfEntries, et.properties)
+
+        val entries = ImmutableList.copyOf(testData.values)
+        val ids = dataApi.createEntities(es.id, entries)
+
+        Assert.assertEquals(0L, dataApi.getEntitySetSize(es.id))
+
+        Thread.sleep(300_000)
```

Comment:
this is how long it takes for that table to get populated?

Reply:
Right now, yes, this is the refresh rate.https://jira.openlattice.com/browse/LATTICE-2003

Reply:
oh yeah. so long :(